docs/reference/objects/accommodation/availabilty_day.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Availability Day
+parent: Accommodation
+---
+
+When using an `availability_day` object you have access to the following attributes.
+
+# availability_day.available
+
+Returns true if the Accommodation is marked as available for a day.
+
+# availability_day.date
+
+Returns the date of the `availability_day`.
```

Comment:
How about the following:

```
The date as a timestamp. This can then be used in conjunction with Liquid's [ built in filters ](https://shopify.github.io/liquid/filters/date/).
```

Reply:
FYI - I've been avoiding using "Returns ..." at the start of every description and just stating what it is instead. e.g "The date ..." vs "Returns the date ...".

I think this reads better and is less repetitive.